low-level/actuators/impl/generic/realstor_encl.py

```diff
@@ -84,13 +84,32 @@ def perform_request(self, jsonMsg):
         response = "N/A"
         try:
             enclosure_request = jsonMsg.get("actuator_request_type").get("storage_enclosure").get("enclosure_request")
-            (request_type, enclosure, component, component_type) = [
-                    s.strip() for s in enclosure_request.split(":")]
+            enclosure_request_data = [
+                s.strip() for s in enclosure_request.split(":")]
+            ctrl_action = ""
+            ctrl_type = ""
+            if enclosure_request_data[-1] == "shutdown":
+                (request_type, _, component, component_type,
+                    ctrl_action) = enclosure_request_data
+            elif enclosure_request_data[-1] == "restart":
+                (request_type, _, component, component_type,
+                    ctrl_type, ctrl_action) = enclosure_request_data
+            else:
+                (request_type, _, component, component_type) = \

```

Comment:
can put paste example of enclosure_request here, just want to see how it look different for GET and control rquests

Reply:
Done